workshop/README.md

```diff
@@ -1 +1,18 @@
-# Kube 101 workshop
+# Istio 101 workshop
+
+## Workshop Setup
+- [Exercise 1 - Accessing a Kubernetes cluster with IBM Cloud Container Service](exercise-1/README.md)
+
+If you are not familar with Kubernetes, we recommend you to follow [Kubernetes 101](https://github.com/IBM/kube101) learn Kubernetes.  If you are already familar with Kubernetes, feel free to proceed to exercises below to learn Istio.
+
+## Creating a Service Mesh with Istio
+
+- [Exercise 2 - Installing Istio](exercise-2/README.md)
```

Comment:
these should be under the "workshop" folder, e.g. ` workshop/exercise-2/README.md`

Reply:
yes, it is planned to be there